roles/tendrl-storage-node/tasks/tendrl-node-agent.yml

```diff
@@ -17,6 +17,7 @@
       line: "graphite_host: {{ graphite_fqdn }}"
   notify:
     - restart tendrl-node-agent
+    - reload rsyslog
```

Comment:
I would rather add this `reload rsyslog` notification to `Install tendrl-node-agent` task. Reloading rsyslog config is needed because of default config file shipped in the package, and has nothing to do with it's configuration.

Reply:
Sure, i will do it

Reply:
Good point.